Makefile

```diff
@@ -13,14 +13,14 @@ prepare-v11:
 	go mod vendor
 
 test:
-	# Skip integration tests in ./test/
-	GO111MODULE=on go test -mod=vendor -tags=$(BUILDTAGS) `go list -mod=vendor -tags=$(BUILDTAGS) ./... | grep -v terraform-validator/test`
+	# Skip integration tests in ./test/ using -short flag
+	GO111MODULE=on go test -short -mod=vendor -tags=$(BUILDTAGS) ./...
```

Comment:
Coes `-mod=vendor` need to be the first argument still?

Reply:
Now I have more data in the build pipeline, I think this is not true and is probably misobservation from myself with some other changes that I have missed.